samples/bootloader/README.rst

```diff
@@ -127,7 +136,7 @@ Complete the following steps to add the bootloader sample as child image to your
 #. Run ``menuconfig`` on your application to enable Secure Boot:
 
    a. Select :guilabel:`Project` -> :guilabel:`Configure nRF Connect SDK project`.
-   #. Go to :guilabel:`Modules` -> :guilabel:`Nordic nRF Connect` and select :guilabel:`Use Secure Bootloader` to enable :option:`CONFIG_SECURE_BOOT`.
+   #. Go to :guilabel:`Modules` -> :guilabel:`Nordic nRF Connect` -> :guilabel:`Bootloader` and set :guilabel:`Use Secure Bootloader` to enable :option:`CONFIG_SECURE_BOOT`.
```

Comment:
Why the change from select to set..?
I realize that I havent't seen how this actually looks like (a screenshot could probably help). Then it's more difficult to give an advice.

Reply:
It is a checkbox. So you either set or unset it.
I think that select is better with radio buttons.

Reply:
Then I totally agree with you. Set, check or tick should be used rather than select for checkboxes :)